apps/web/src/components/Feed/FeedList.tsx

```diff
@@ -159,73 +69,91 @@ export default function FeedList({
     setIsLoading(false);
   }, [loadNextPage]);
 
-  useEffect(
-    function recalculateHeightsWhenEventsChange() {
-      virtualizedListRef.current?.recomputeRowHeights();
-    },
-    [feedData, measurerCache]
-  );
+  useEffect(() => {
+    const [lastItem] = [...feeds].reverse();
+
+    if (!lastItem) {
+      return;
+    }
+
+    if (lastItem.index >= feedData.length - 1 && hasNext && !isLoading) {
+      handleLoadMoreClick();
+    }
+  }, [feedData.length, feeds, hasNext, isLoading, handleLoadMoreClick]);
```

Comment:
Pretty sure we don't want infinite scroll for the feed right?

Reply:
we actually do now @tbezman !